queue_job/delay.py

```diff
@@ -607,14 +647,21 @@ def __init__(
             channel=channel,
             identity_key=identity_key,
         )
+        self.split = split
 
     @property
     def recordset(self):
         return self.delayable.recordset
 
     def __getattr__(self, name):
         def _delay_delayable(*args, **kwargs):
-            getattr(self.delayable, name)(*args, **kwargs).delay()
+            delayable = getattr(self.delayable, name)(*args, **kwargs)
+            if self.split:
+                group = delayable.split(self.split)
+                group.delay()
+                return [d._generated_job for d in group._delayables]
+
+            delayable.delay()
```

Comment:
Is this necessary? Couldn't a developer just call `delayable.split()` themselves?

Reply:
Removed